website/docs/flink-quick-start-guide.md

```diff
@@ -287,10 +289,37 @@ Refers to [Table types and queries](/docs/concepts#table-types--queries) for mor
 
 This is similar to inserting new data.
 
+<Tabs
+defaultValue="flinksql"
+values={[
+{ label: 'Flink SQL', value: 'flinksql', },
+{ label: 'DataStream API', value: 'dataStream', },
+]}
+>
+
+<TabItem value="flinksql">
+
+Creates a Flink Hudi table first and insert data into the Hudi table using SQL `VALUES` as below.
+
 ```sql
+-- Update Queries only works with batch execution mode
 SET 'execution.runtime-mode' = 'batch';
 UPDATE hudi_table SET fare = 25.0 WHERE uuid = '334e26e9-8355-45cc-97c6-c31daf0df330';
 ```
+</TabItem>
+
+<TabItem value="dataStream">
+
+Creates a Flink Hudi table first and insert data into the Hudi table using DataStream API as below.
+When new rows with the same primary key arrive in stream, then it will be be updated.
+In the insert example incoming row with same record id will be updated.
+
+Refer Update Example [here](https://github.com/ad1happy2go/hudi-examples/blob/main/flink/src/main/java/com/hudi/flink/quickstart/HudiDataStreamReader.java)
+
+</TabItem>
+
+</Tabs
+>
 
 Notice that the save mode is now `Append`. In general, always use append mode unless you are trying to create the table for the first time.
```

Comment:
Its not clear where the save mode is. Is this carried over from Spark quickstart by any chance ?